lib/s3middleware/objectLegalHold.js

```diff
@@ -0,0 +1,117 @@
+const { parseString } = require('xml2js');
+const errors = require('../errors');
+
+/*
+    Format of the xml request:
+    <LegalHold>
+        <Status>ON|OFF</Status>
+    </LegalHold>
+*/
+
+/**
+ * validate status - validate status xml
+ * @param {object} status - parsed status object
+ * @return {object} - contains validated status object on success,
+ * error on failure
+ */
+function _validateStatus(status) {
+    const validatedStatus = {};
+    const expectedValues = ['OFF', 'ON'];
```

Comment:
Have you checked if `aws-cli` or `aws-sdk` allows lowecase/mixed-case usage of `ON/OFF`?

Reply:
It doesn't allow; I checked by passing  `On`, `off`, `Off`, `on` as the Status via the SDK and got 
```'MalformedXML: Status request xml must be one of "OFF", "ON"\n    at Request.extractError```